.github/workflows/build-publish-lib.yml

```diff
@@ -20,79 +29,47 @@ jobs:
           architecture: x64
           registry-url: "https://registry.npmjs.org"
 
-      - name: Get version
-        id: package-version
-        uses: martinbeentjes/npm-get-version-action@v1.1.0
-
-      - name: Check tag does not exist yet
-        run: if git rev-list v${{ steps.package-version.outputs.current-version }}; then echo "Tag already exists. Aborting the release process."; exit 1; fi
-
-      - name: Tag commit and push
-        uses: mathieudutour/github-tag-action@v5.6
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          custom_tag: ${{ steps.package-version.outputs.current-version }}
-
-      - name: Install and link 🔗
+      - name: Install and Link 🔗
         run: |
           npm ci
           npm link
           npm link @maplibre/maplibre-gl-directions
 
-      - name: Prepare release
-        id: prepare_release
-        run: |
-          echo ::set-output name=version_tag::v${{ steps.package-version.outputs.current-version }}
-          echo ::set-output name=release_type::$(node -e "console.log(require('semver').prerelease('${{ steps.package-version.outputs.current-version }}') ? 'prerelease' : 'regular')")
-
       - name: Build 🔧
         run: |
           npm run build
 
-      - name: Publish NPM package (regular)
-        if: ${{ steps.prepare_release.outputs.release_type == 'regular' }}
+      - name: Bump Version
         run: |
-          npm publish --access public --non-interactive
+          git config --global user.email "g.smellyshovel@gmail.com"
+          git config --global user.name "Matthew Mamonov"
```

Comment:
The workflow could be started by any user with the correct permissions. Can you use the config suggested here: https://github.com/actions/checkout#push-a-commit-using-the-built-in-token

Reply:
Done